panoramix/assets/html/basic.html

```diff
@@ -0,0 +1,38 @@
+<html>
```

Comment:
I don't think this is the most up-to-date version of this file, `app_name ` is undefined and a lot of these css paths are off (e.g., with select2 you should see errors after you run npm install and decrement versions). can you replace it with the current templates/basic.html ?